db/schema.rb

```diff
@@ -20,8 +20,8 @@
     t.datetime "updated_at"
     t.float    "latitude"
     t.float    "longitude"
-    t.string   "address",    limit: 255
-    t.boolean  "reviewed",               default: false
+    t.string   "address"
```

Comment:
I didn't add any migrations, so someone must have not migrated before they added the schema.